bin/runci.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
```

Comment:
In language-server we avoid this script with https://github.com/scalameta/language-server/blob/87452e193986b784d7e0bad1fc2387997fa42e92/.travis.yml#L10-L19

Reply:
Okay, i'll do the same